### ISSUE TITLE
Errors on scans on FreeBMD2

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#287]
FreeBMD1 text on scan page: If the scan does not contain the entry you may be able to find the scan yourself by clicking here. If the scan does not contain the entry you can leave feedback by clicking here.

On Entry Information Page, below scans, text now needed, suggest:

If the scan does not contain the entry you may be able to find the scan yourself by clicking here [links to FreeBMD1? or ??? - page is (e.g.) https://www.freebmd.org.uk/cgi/scan-response.pl?y=1838&q=1&e=1&a=find&d=bmd_1600692744&r=IANSON%7CJohn%7CSt%20Giles%7C1%7C30&signature=zjeTvn0aNjxG0%2BmyQv4l6w&extra=ANC-01,340&extra=LDS%2F1838b1-I-0340.jpg,340. If the scan does not contain the entry you can leave feedback by clicking here 

That takes you to a page with text (e.g. )
Are you sure that record
IANSON,John,St Giles,1,30
is not in scan
ANC-01/1838B1-I-0340.jpg
Please do not use rejection to indicate that the record has been incorrectly transcribed, rather use the correction process on the Information page.
- with a 'confirm button.

For Decision: do we create this process on FreeBMD2 for MVP, or simply link through to FreeBMD1 at this point (sending the information needed to generate the correct data in the links?

Depending on decision, we may need to use this text https://docs.google.com/document/d/17EzCa8Gl130ACmG00LEJFKcAU-83Yg2Z8Oiu0Cu0BeU/edit?usp=sharing

**Comment by richpomfret** [comment]
@Vino-S to investigate/explore options here.

**Comment by Vino-S** [comment]
Finding missing scan takes us to the FreeBMD image server(https://images.freebmd.org.uk/cgi/choose.pl) so we need to provide a link for the image server from BMD2

**Comment by Vino-S** [comment]
@Vino-S to update the story once code move to vervet

**Comment by Vino-S** [comment]
ready for testing in test2

**Comment by PatReynolds** [comment]
I can't see any difference on test2, @Vino-S  - ![image.png](https://images.zenhubusercontent.com/5b695a5dac3a200155c765e3/147a9a15-9440-470b-b674-1f6d48c8107d)

**Comment by DeniseColbert** [comment]
@Vino-S needs more info on what difference is expected

**Comment by richpomfret** [comment]
@PatReynolds to take a look and clarify for Vino.

**Comment by PatReynolds** [comment]
I have made a document showing what I see on test2, and what I expected to see: https://docs.google.com/document/d/1WOZVdbxCbdXtl0BFxyHv_99zdLv05FwZvcwziItS8vo/edit?usp=sharing

**Comment by richpomfret** [comment]
@Vino-S to take a look and see if the requirement is now clear.

**Comment by Vino-S** [comment]
Improved display of scans in test3
Record Example with scan:
1. First Name: Martha, Surname: TASKER, Volume: 19 and Page: 34

**Comment by PatReynolds** [comment]
Please ignore previous comment .... 
What currently happens on FreeBMD1:

There is a button to report an error, which adds this text:
![image.png](https://images.zenhubusercontent.com/5b695a5dac3a200155c765e3/d0841931-7811-4233-936d-ee9d1686283c)

Need to check with FreeBMD1 team what of this functionality is needed.

**Comment by PatReynolds** [comment]
Wrote to Allan and Barrie 25 Sep 2021, but realise that this is about reporting errors on scans, not missing scans (the subject of this issue).

**Comment by DeniseColbert** [comment]
@Vino-S to check FreeBMD1 to see if we need to add a statement about how to report missing scans (Report a Data Problem button)

**Comment by Vino-S** [comment]
In FreeBMD1 researcher can provide feedback if the Scan has the searched record or they can reject the Scan.
![image.png](https://images.zenhubusercontent.com/5ca2316a45147541ad68a420/f2d57641-6ed8-403d-a34f-b189c974a40a)

**Comment by richpomfret** [comment]
Agreed to go with ranking via BMD1 site for now and hold off on any additional features/implementation of this on BMD2 until post MVP. To check on product requirements list.
